### PR TITLE
Common normalizers

### PR DESCRIPTION
**Pull request by jbabe**
- A few extra normalizers that we commonly use across apps

### CODE REVIEW

File: attribute_normalizer-extras.gemspec [hunk 1]
```diff
@@ -7,9 +7,9 @@ Gem::Specification.new do |spec|
   spec.name          = "attribute_normalizer-extras"
   spec.version       = AttributeNormalizer::Extras::VERSION
   spec.authors       = ["Jordan Babe"]
-  spec.email         = ["jordan.babe@ama.ab.ca"]
-  spec.summary       = %q{TODO: Write a short summary. Required.}
-  spec.description   = %q{TODO: Write a longer description. Optional.}
+  spec.email         = ["jorbabe@gmail.com"]
```
Comment (ryanjones) [inline attribute_normalizer-extras.gemspec]: Needs all AMA GH accounts.

File: lib/attribute_normalizer/extras.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
 
-module AttributeNormalizer
-  module Extras
-    # Your code goes here...
-  end
-end
+extras = {
+ gsub:  AttributeNormalizer::Normalizers::GsubNormalizer,
+ postal_code: AttributeNormalizer::Normalizers::PostalCodeNormalizer,
+ squash: AttributeNormalizer::Normalizers::SquashNormalizer
```
Comment (ryanjones) [inline lib/attribute_normalizer/extras.rb]: Can we make this name more descriptive?

Comment (MathieuGilbert) [inline lib/attribute_normalizer/extras.rb]: `space_stripper`

Comment (jbabe) [inline lib/attribute_normalizer/extras.rb]: the original gem has a `:squish` and a `:whitespace`

Comment (jbabe) [inline lib/attribute_normalizer/extras.rb]: so i wanted it to be called something different from both - if only there was a verb for removing all whitespace characters from a string

**Comment by jbabe** [comment]
:wave:

**Comment by ryanjones** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**